framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.16 under development
 ------------------------
 
+- Bug #16763: Fix Unable to load dynamic library pdo_sqlsrv.so (marcelodeandrade)
```

Comment:
Please do not add changelog entries about tests/documentation fixes - changelog is about changes in framework, anything else is only a noise from end user perspective.